rest_framework_simplejwt/backends.py

```diff
@@ -16,14 +16,25 @@
 
 
 class TokenBackend:
-    def __init__(self, algorithm, signing_key=None, verifying_key=None, audience=None, issuer=None):
+    def __init__(
+        self,
+        algorithm,
+        signing_key=None,
+        verifying_key=None,
+        audience=None,
+        issuer=None,
+        jwk_url=None,
```

Comment:
Should help with type checking so you don't need to write `if jwt_url is not None`
```suggestion
        jwk_url: str = None,
```